app/views/sessions/welcome.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Welcome</h1>
+<% if logged_in? %>
```

Comment:
If a user is logged in then maybe they could go straight into the posts page rather than having to click to view the posts

Reply:
We actually did modify for it to do that I think from our latest commits after this.